DataFormats/Scouting/src/classes_def.xml

```diff
@@ -33,22 +33,27 @@
   <class name="Run3ScoutingPFJet" ClassVersion="3">
       <version ClassVersion="3" checksum="3819751468"/>
   </class>
-  <class name="Run3ScoutingParticle" ClassVersion="4">
+  <class name="Run3ScoutingParticle" ClassVersion="7">
+   <version ClassVersion="7" checksum="2425016900"/>
+   <version ClassVersion="6" checksum="3844610129"/>
+   <version ClassVersion="5" checksum="583559071"/>
```

Comment:
Please drop versions 5 and 6, rename 7 as 5, move it below 4, and use the same indentation.